content/developer/howtos/standalone_owl_application.rst

```diff
@@ -0,0 +1,171 @@
+    <templates xml:space="preserve">
+        <t t-name="your_module.Root">
+            Hello, World!
+        </t>
+    </templates>
+
+Then create the JavaScript file for that component in `/your_module/static/src/standalone_app/root.js`.
+
+.. code-block:: js
+
+    /** @odoo-module */
+    import { Component } from "@odoo/owl";
+
+    export class Root extends Component {
+        static template = "your_module.Root";
+        static props = {};
+    }
+
+.. seealso::
+   Owl components :ref:`reference<frontend/components>`.
+
+2. Setting up the environment and mounting the application
+==========================================================
+
+An Owl application needs an environment, and the Odoo JavaScript framework needs that environment to
+contain the :ref:`services<frontend/services>`.
+Starting the services is also required to load the translations, which we need to do before mounting
+our Owl application, so that we can give Owl a working translation function.
+
+Create the JavaScript file that will mount the app in `/your_module/static/src/standalone_app/app.js`.
+
+.. code-block:: js
+
+    /** @odoo-module */
+    import { whenReady } from "@odoo/owl";
+    import { mountComponent } from "@web/env";
+    import { Root } from "./root";
+
+    whenReady(() => mountComponent(Root, document.body));
+
+
+3. Creating an assets bundle containing our code
+================================================
+
+In the manifest of your module, create a new :ref:`assets bundle<reference/assets_bundle>`.
+It should include the `web._assets_core` bundle, which contains the Odoo JavaScript
+framework and the core libraries it needs (e.g. Owl and luxon), after which you can have a
+glob that adds all the files for your application in the bundle.
+
+.. code-block:: py
+
+    {
+        # ...
+        'assets': {
+            'your_module.assets_standalone_app': [
+                ('include', 'web._assets_helpers'),
+                'web/static/src/scss/pre_variables.scss',
+                'web/static/lib/bootstrap/scss/_variables.scss',
+                ('include', 'web._assets_bootstrap'),
+                ('include', 'web._assets_core'),
+                'your_module/static/src/standalone_app/**/*',
+            ],
+        }
+    }
+
+The lines before `web._assets_core` are bundles and scss files that are required to make
+Bootstrap work. They are required, as the components of the web framework use bootstrap classes for
+their styling and layout.
+
+.. note::
+    Make sure that the files for your standalone app are only added to this bundle, if you already
+    have a definition for `web.assets_backend` or `web.assets_frontend` and they have globs, make
+    sure these globs don't match the files for your standalone app, otherwise the startup code for
+    your app will conflict with the existing startup code in those bundles.
+
+.. seealso::
+    :ref:`module manifest reference<reference/module/manifest>`.
```

Comment:
Use the `doc` role to link entire pages.
```suggestion
    :doc:`Module manifests reference </developer/reference/backend/module>`
```

Reply:
Definitely want to point to the manifest section here, if another section gets added on top of the manifest section this would be broken.